### PR TITLE
Ticket 4287: update from tests

### PR DESCRIPTION
**Pull request by DominicOram**
See https://github.com/ISISComputingGroup/IBEX/issues/4287

### CODE REVIEW

File: tests/rotsc.py [hunk 1]
```diff
@@ -63,3 +85,60 @@ def test_GIVEN_current_position_WHEN_position_set_to_current_position_THEN_setpo
         # WHEN
```
Comment (ChrisColeExpControl) [inline tests/rotsc.py:85]: This test is now failing consistently on my machine due to the new log messages being generated when switching states, as assert_pv_processed is looking for a log message which gets split up by them.
Failure is happening on line 81 but github won't let me comment there.

Comment (DominicOram) [reply tests/rotsc.py:85]: Annoyingly I can't reproduce this on mine. Can I see your `log_ROTSC_01_devsim_ioc.log`?

Comment (DominicOram) [reply tests/rotsc.py:85]: Ok, think I've updated the process checking to be more robust

Comment (DominicOram) [reply tests/rotsc.py:85]: Can you try it now along with https://github.com/ISISComputingGroup/genie_python/pull/245?

**Pull request merged**